salt/utils/vault.py

```diff
@@ -310,6 +322,17 @@ def make_request(
             pass
     url = "{0}/{1}".format(vault_url, resource)
     headers = {"X-Vault-Token": str(token), "Content-Type": "application/json"}
+    # Vault Enterprise requires a namespace
+    if "namespace" in args:
+        namespace = args["namespace"]
+    else:
+        try:
+            namespace = __opts__.get("vault").get("namespace", None)
+        except (TypeError, AttributeError):
+            # Don't worry about setting verify if it doesn't exist
+            pass 
+    if namespace is not None:
```

Comment:
You'll get `"namespace"` is not defined exception here iv `"vault"` not in `__opts__`.